packages/pigeon/lib/pigeon_lib.dart

```diff
@@ -544,9 +551,6 @@ class _RootBuilder extends dart_ast_visitor.RecursiveAstVisitor<Object?> {
         .removeWhere((Class x) => !referencedTypeNames.contains(x.name));
 
     final List<Enum> referencedEnums = List<Enum>.from(_enums);
-    referencedEnums.removeWhere(
```

Comment:
Do you want to leave a TODO here to re-add this?

Reply:
Not necessarily.  Only generating output for referenced types has lead to a lot of bugs.  It was necessary when we we were using reflection, now it isn't needed.  If we start supporting multiple imports in pigeon files we might have to revisit it again.